pco/src/chunk_config.rs

```diff
@@ -55,6 +55,9 @@ pub enum FloatMultSpec {
 pub enum FloatQuantSpec {
   #[default]
   Disabled,
+  /// Automatically decide whether to use FloatQuant, based on an estimate of how much it would
+  /// save compared to other available modes.
+  Enabled,
   /// If you know how many bits of precision are unused, you can supply that
   /// number here.
   Provided(Bitlen),
```

Comment:
GH won't let me comment on the line below this but there is a todo on lines 61-62 that it's time to remove